metrics.py

```diff
@@ -42,6 +43,41 @@ def rmse_inf(data_ground_truth, data_predicted):
     return rmse_inf
 
 
+def rmse_inf_tensor(data_ground_truth, data_predicted):
+    """
+    Calculate the Root Mean Squared Error (RMSE) normalized by the infinity norm.
```

Comment:
Add the term of pytorch to make difference between numpy function and pytorch

Reply:
ya lo que tu digas

Reply:
perfect!